servers/lib/server/handlers/createteam.test.coffee

```diff
@@ -14,6 +14,7 @@ KONFIG                              = require 'koding-config-manager'
 { testCsrfToken }                   = require '../../../testhelper/handler'
 { generateRegisterRequestParams }   = require '../../../testhelper/handler/registerhelper'
 { generateCreateTeamRequestParams } = require '../../../testhelper/handler/teamhelper'
+{ Status }                          = require '../../../../client/app/lib/redux/modules/payment/constants'
```

Comment:
woo–hoo using client side code on backend :D 

I dont know if it is good or bad tho in general 😅